openwisp_utils/api/serializers.py

```diff
@@ -0,0 +1,14 @@
+from django.core.exceptions import ImproperlyConfigured
+
+try:
+    from rest_framework import serializers
+except ImportError:  # pragma: nocover
+    raise ImproperlyConfigured(
+        'Django REST Framework is required to use this feature but it is not installed')
```

Comment:
we usually break this kind of code into multiple lines with something like:

```python
raise ImproperlyConfigured('Django REST Framework is required to use '
                           'this feature but it is not installed')
```

Could you update these 2 lines for consistency reasons please?